javascript/canvas-zoom.js

```diff
@@ -955,9 +964,13 @@
 
         const shouldHideButtons = window.isDrawing && hotkeysConfig.canvas_zoom_hide_btn;
 
-        if (shouldHideButtons !== buttonsHidden) {
-          hideCanvasButtons(shouldHideButtons);
-          buttonsHidden = shouldHideButtons;
+        try {
```

Comment:
For some reason there was a bug, applied a small hack that would not have errors